src/utils/Sorter.js

```diff
@@ -555,16 +555,20 @@ export default Backbone.View.extend({
     if (isTextNode(el)) return;
     const style = el.style || {};
     const $el = $(el);
-    const $parent = parent && $(parent);
+    let $parent = parent && $(parent);
+    if(parent.shadowRoot){
+      const slot = parent.shadowRoot.querySelector('slot');
+      if(slot) $parent = $(slot).css('display') === 'contents'?$(slot.parentElement):$(slot);
```

Comment:
Are you able to illustrate with an example such a case? I mean, is it valid for general use of Web Components or it fixes only your case?